README.md

```diff
@@ -129,9 +129,9 @@ Expected output:
 
 ## Removing Deployments
 
-`helm del --purge <release-name>`
+`helm uninstall <release-name>`
```

Comment:
Please use `helm del <release-name>` <-- this will match the deployment guide changes for helm v3.